iocore/net/UnixUDPNet.cc

```diff
@@ -534,7 +534,7 @@ UDPNetProcessor::recvfrom_re(Continuation *cont, void *token, int fd, struct soc
     cont->handleEvent(NET_EVENT_DATAGRAM_READ_COMPLETE, event);
     completionUtil::destroy(event);
     return ACTION_RESULT_DONE;
-  } else if (actual == 0 || (actual < 0 && actual == -EAGAIN)) {
+  } else if (actual == 0 || (actual < 0 && actual == -EAGAIN)) { // lgtm[cpp/constant-comparison]
```

Comment:
Seems like this should simply be:
```
  } else if (actual == 0 || actual == -EAGAIN) {
```

Reply:
Ah, right. Now I got what the alert says.